lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -524,19 +544,42 @@ export default class BitcoinProcessor {
   }
 
   /**
-   * Return proof-of-fee value for a particular block.
+   * Modifies the given array and update the normalized fees, then write to block metadata store.
+   * @param blocks the ordered block metadata to set the normalized fee for.
    */
-  public async getNormalizedFee (block: number): Promise<TransactionFeeModel> {
+  private async writeBlocksToMetadataStoreWithFee (blocks: BlockMetadataWithoutNormalizedFee[]) {
+    for (const block of blocks) {
+      const normalizedFee = (await this.getNormalizedFee(block.height)).normalizedTransactionFee;
```

Comment:
Should we file a improvement issue so that we have `getNormalizedFees` so that we make one range query to DB instead of many?

Reply:
Opened issue
https://github.com/decentralized-identity/sidetree/issues/936